experiment_prototype/experiment_prototype.py

```diff
@@ -192,11 +192,10 @@
     in this function. The default is None if no function handle is supplied.
 
     encode_fn(beam_iter, sequence_num, num_pulses, num_samples):
-        return np.ones(size=(num_pulses, num_samples))
+        return np.ones(size=(num_pulses))
```

Comment:
Can probably modify lines 189-192 as well to remove any reference to 'phases' instead of 'phase'

Reply:
I should actually change this too so that num_samples isn't a required argument, since there is no need for it anymore.